README.md

```diff
@@ -1,4 +1,9 @@
-# GEMP-UFPA
+<p align="center">
+  <img width="600" src="images/logo-without-text.png">
+  <h1 align="center">GEMP-UFPA</h1>
+  <h2 align="center">Grupo de Estudos Para a Maratona de Programação - UFPA</h2>
```

Comment:
Que tal se aumentássemos o nome "GEMP - UFPA", e retirássemos o nome por extenso que tá logo embaixo? Porque esse nome por extenso já aparece na descrição, então para alguém que for ler do que se trata o projeto vai acabar sendo muito repetitivo. Poderíamos colocar um negrito dentro da descrição para destacar mais.

Reply:
Acho válido, principalmente se conseguirmos retirar essas várias linhas que separam os textos.